app/config/config_test.yml

```diff
@@ -3,6 +3,7 @@
 
 imports:
     - { resource: config.yml }
+    - { resource: secret_parameters.php }
```

Comment:
I would prefer to add configuration with such parameters not the "hacky hack" that uses envs